### ISSUE TITLE
Scalable for fitting larger dataset

### ISSUE THREAD

**Issue by NNDam** [lyndonzheng/Pluralistic-Inpainting#67]
Thank you for the amazing work !!! I've trained your model with my custom dataset (about 40k face images) and  it worked well, but i found that loss plot wasn't reduce (like other loss function, CE for example) for large amount of iters and your default model very small (9 Mb). So the question is what 's the reason, by model architecture ? if reason is model architecture, how can i scale up for fitting well larger dataset.
Thank you !

**Comment by lyndonzheng** [comment]
Hi @NNDam, the loss plot is not reduced due to the adversarial learning, which does not have a straightforward loss function. Our lab has limited GPU, so we only designed a simple model to demonstrate our "pluralistic" idea, rater than a larger model for longer training.
